eng/versioning/version_client.txt

```diff
@@ -6,7 +6,7 @@ com.azure:azure-sdk-parent;1.6.0;1.6.0
 com.azure:azure-client-sdk-parent;1.7.0;1.7.0
 com.azure:azure-ai-textanalytics;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-core;1.3.0;1.4.0-beta.1
-com.azure:azure-core-amqp;1.0.1;1.1.0-beta.1
+com.azure:azure-core-amqp;1.0.2;1.0.2
```

Comment:
You don't need to modify the dependency version, after we release, Jim's bot will go and swap current to dependency version then set the new "current" version.

Reply:
What @conniey said is correct. The dependency version of com.azure:azure-core-amqp is 1.0.1 and should remain 1.0.1 until 1.0.2 has been released.